scripts/install-deps.sh

```diff
@@ -293,6 +300,7 @@ do_install_catch2() {
     mkdir cmake || return
     pushd cmake > /dev/null || return
 
+    export CC=/usr/bin/gcc-10 CXX=/usr/bin/g++-10
```

Comment:
as above

Reply:
I'll forgo the export and inline on the cmake line, which will not modify env externally

Reply:
See https://github.com/Azure/iot-hub-device-update/pull/516/commits/05cb632c4add1136d241c7deb0829a942e54efa5